sds_data_manager/lambda_code/efs_lambda/lambda_function.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os
+from pathlib import Path
+
+import boto3
+
+# Define the paths
+mount_path = os.environ.get("EFS_MOUNT_PATH")
+
+attitude_symlink_path = f"{mount_path}/latest_attitude_kernel.ah.a"
+ephemeris_symlink_path = f"{mount_path}/latest_ephemeris_kernel.bsp"
+
+
+def create_symlink(source_path: str, destination_path: str) -> None:
```

Comment:
```suggestion
def create_symlink(source_path: Path, destination_path: Path) -> None:
```
If you change over to `Path` objects early (even in the string paths globally above this) you could use these directly down below without having to create the new Path objects.